components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt.ui/src/main/resources/web/identity-claim-mgt/add-local-claim.jsp

```diff
@@ -176,26 +176,26 @@
                 function validate() {
 
                     var value = document.getElementsByName("localClaimURI")[0].value;
-                    if (value == '') {
-                        CARBON.showWarningDialog('<fmt:message key="claim.uri.is.required"/>');
+                    if (value.trim() == '') {
```

Comment:
Remove unused,

- claim.uri.is.required
- displayname.is.required
- description.is.required